src/components/10-atoms/button-link/index.scss

```diff
@@ -21,7 +21,7 @@
   border-bottom-color: $color-deep-sapphire;
 
   @include breakpoint($mediaquery-sm-up) {
-    @include _text(default);
+    min-height: 40px;
```

Comment:
Why did you add min-height?

Reply:
cant answer you...since the merge conflicts everything changed :(

Reply:
There has nothing changed as you can see in the diffing  `src/components/10-atoms/button-link/index.scss`